src/components/common/Footer/index.tsx

```diff
@@ -14,7 +17,7 @@ function Footer(): JSX.Element {
               .flatMap((categoryInfo) => categoryInfo.submenu)
               .map((subMenuInfo) => (
                 <li className={styles.footer__service} key={subMenuInfo.title}>
-                  <Link to={subMenuInfo.link}>
+                  <Link to={subMenuInfo.link} onClick={() => logger.actionEventClick({ actionTitle: 'CAMPUS', title: 'footer_bus', value: '버스' })}>
```

Comment:
title을 `footer`로 하고 전체에 로깅을 심는게 맞는 거 같은데 어떻게 생각하시나요?
```suggestion
                  <Link to={subMenuInfo.link} onClick={() => logger.actionEventClick({ actionTitle: 'CAMPUS', title: 'footer', value: 'subMenuInfo.title' })}>
```

Reply:
맞네요 ㅋㅋㅋ 요렇게 하면 버스밖에 로깅이 안 심기는게 맞습니당
적절하게 수정했습니다👍